projects/fsastorefrontlib/src/cms-components/checkout/components/cart-coupon/cart-coupon.component.html

```diff
@@ -6,23 +6,27 @@
     <div class="form-group">
       <form [formGroup]="couponForm">
         <div class="cx-cart-coupon-container row">
-          <div class="col-6">
-            <input
-              type="text"
-              class="form-control input-coupon-code"
-              formControlName="couponCode"
-            />
-          </div>
-          <div class="col-6">
-            <button
-              (click)="applyVoucher()"
-              class="primary-button btn-block"
-              type="button"
-              [disabled]="cartIsLoading$ | async"
-              [class.disabled]="cartIsLoading$ | async"
-            >
-              {{ 'voucher.apply' | cxTranslate }}
-            </button>
+          <div class="col-12">
+            <div class="input-group">
+              <input
+                type="text"
+                class="form-control input-coupon-code"
+                formControlName="couponCode"
+                aria-describedby="apply-voucher"
+              />
+              <div class="input-group-append w-25">
```

Comment:
Why do we need w-25? We should avoid using these classes for setting sizes since they have `!important` declaration in them, and are hard to override eventually.

Reply:
Because of append button contains a font-awesome icon that is short, so I need to set some width. I'll remove BS class and set it as CSS rule